src/main/java/io/pravega/perf/ReaderWorker.java

```diff
@@ -100,6 +100,7 @@ public void EventsReaderRW() throws IOException {
                     i++;
                 }
             }
+        } catch(WorkerCompleteException ignore) {
```

Comment:
No, because the Exception only exists to indicate to the logic that the reader has completed, it's not an actual exception or an error condition.  

The `PravegaBatchReaderWorker` (which is the only worker that throws this exception) will log that it competed everything, so another log would just be redundant.